app/views/hyrax/admin/show.html.erb

```diff
@@ -36,4 +36,20 @@
       </div>
     </div>
 </div>
+
 <%= render 'collections' %>
+
+<div class="row">
```

Comment:
I don't think I can test this as it goes into a CDATA block and thus isn't parsed by capybara.